torch/special/__init__.py

```diff
@@ -15,15 +16,81 @@
     \text{out}_{i} = \ln \Gamma(|\text{input}_{i}|)
 """ + """
 Args:
-    input (Tensor): the input tensor.
+    {input}
 
 Keyword args:
-    out (Tensor, optional): the output tensor.
+    {out}
 
 Example::
 
     >>> a = torch.arange(0.5, 2, 0.5)
     >>> torch.special.gammaln(a)
     tensor([ 0.5724,  0.0000, -0.1208])
 
-""")
+""".format(**common_args))
+
+erf = _add_docstr(_special.special_erf,
+                  r"""
+erf(input, *, out=None) -> Tensor
+
+Computes the error function of :attr:`input`. The error function is defined as follows:
+
+.. math::
+    \mathrm{erf}(x) = \frac{2}{\sqrt{\pi}} \int_{0}^{x} e^{-t^2} dt
+""" + r"""
+Args:
+    {input}
+
+Keyword args:
+    {out}
+
+Example::
+
+    >>> torch.erf(torch.tensor([0, -1., 10.]))
+    tensor([ 0.0000, -0.8427,  1.0000])
+""".format(**common_args))
+
+erfc = _add_docstr(_special.special_erfc,
+                   r"""
+erfc(input, *, out=None) -> Tensor
+
+Computes the complementary error function of :attr:`input`.
+The complementary error function is defined as follows:
+
+.. math::
+    \mathrm{erfc}(x) = 1 - \frac{2}{\sqrt{\pi}} \int_{0}^{x} e^{-t^2} dt
+""" + r"""
+Args:
+    {input}
+
+Keyword args:
+    {out}
+
+Example::
+
+    >>> torch.erfc(torch.tensor([0, -1., 10.]))
+    tensor([ 1.0000, 1.8427,  0.0000])
+""".format(**common_args))
+
+erfinv = _add_docstr(_special.special_erfinv,
+                     r"""
+erfinv(input, *, out=None) -> Tensor
+
+Computes the inverse error function of :attr:`input`.
+The inverse error function is defined in the range :math:`(-1, 1)` as:
```

Comment:
In a docs cleanup PR this may want to elaborate on how the inverse function is computed, and why its range is restricted (and what happens when inputs beyond that range are given).